dev/0.utf

```diff
@@ -267411,6 +267411,3 @@ if %achieve_1 == 1 && %achieve_2 == 1 steamsetachieve "UMI47"
 
 
 return
-
```

Comment:
it doesn't make any differenc,e but what are these randomyl deleted lines?

in git (git gui or otherwise) you can select exactly what parts you want as part of the commit, I would suggest removing this unless you really meant to delete these lines

Reply:
Sorry -- I didn't mean to do this as part of this commit, my editor just deletes extraneous empty lines at the end of the file when I save the file. If you'd like, I can return these -- though I am not sure if there's any need?

Reply:
oh ok, if it's justt at the end of the file it's fine, they shouldn't be there anyway.